public/js/app.js

```diff
@@ -1,56 +1,58 @@
 'use strict';
 
-function Project(rawDataObj) {
-  this.name = rawDataObj.name;
-  this.url = rawDataObj.url;
-  this.about = rawDataObj.about;
-  this.image = rawDataObj.image;
-  Project.all.push(this);
-}
-
-
-Project.fetchAll = function() {
-  navHandler();
-  if (localStorage.rawData) {
-    Project.initializeProjects(JSON.parse(localStorage.rawData));
-  } else {
-    $.get('data/projects.json', function(response) {
-      localStorage.setItem('rawData', JSON.stringify(response))
-      Project.initializeProjects(response);
-    })
+let app = {};
```

Comment:
You should add `app` to this assignment, like in the lab, for when you add more script files. Just in case.